schema.sql

```diff
@@ -0,0 +1,51 @@
+-- File to create database and tables within the database
+
+DROP DATABASE IF EXISTS plants;
+CREATE DATABASE plants;
+\c plants;
+
+CREATE TABLE IF NOT EXISTS sunlight_type (
+    sunlight_type_id INT UNIQUE NOT NULL,
+    sunlight_type TEXT NOT NULL,
+    PRIMARY KEY (sunlight_type_id)
+);
+
+CREATE TABLE IF NOT EXISTS plant_origin (
+    plant_origin_id INT GENERATED ALWAYS AS IDENTITY,
+    latitude SMALLINT NOT NULL,
+    longitude SMALLINT NOT NULL,
+    country TEXT NOT NULL,
+    PRIMARY KEY (plant_origin_id)
+);
+
+CREATE TABLE IF NOT EXISTS botanist (
+    botanist_id INT GENERATED ALWAYS AS IDENTITY,
+    botanist_name TEXT NOT NULL,
+    botanist_email TEXT,
+    botanist_phone_number TEXT,
+    PRIMARY KEY (botanist_id)
+);
+
+CREATE TABLE IF NOT EXISTS plant (
+    plant_id SMALLINT NOT NULL UNIQUE,
+    plant_name TEXT NOT NULL,
+    plant_scientific_name TEXT, 
+    plant_origin SMALLINT,
+    last_watered TIMESTAMP, 
+    botanist_id SMALLINT NOT NULL,
```

Comment:
Are we sure whether the botanist is unique to each plant or just unique to each reading? I think this affects whether `botanist_id` exists as a column in the `plant` table or the `reading_information` table